recordedit/recordEdit.app.js

```diff
@@ -183,6 +183,15 @@
                 } else {
                     if ($rootScope.reference.canCreate) {
                         $rootScope.displayname = $rootScope.reference.displayname;
+
+                        // populate defaults
+                        angular.forEach($rootScope.reference.columns, function(column) {
+                            // typeof column.default !== "function"
+                            if (column.default) {
```

Comment:
Hmm I see a comment in your tests about adding "tests for default values for foreign keys when implemented" so perhaps my comment above is premature.